### ISSUE TITLE
Custom Vault containers by IDs

### ISSUE THREAD

**Issue by TomPiXX** [MinecraftWars/Gringotts#118]
It would be nice to have custom containers as Vault.
Maybe we could just set them with the block IDs so other plugins / mods would be supported.

**Comment by jastice** [comment]
This would probably not be too hard and I'll gladly accept pull requests for it ;)

**Comment by hunter688** [comment]
Yes please, add this feature. I think items in the modded chests ale stored as NBT.

**Comment by avaloon** [comment]
some new for this request ?
